icu4c/source/data/buildtool/request_types.py

```diff
@@ -303,8 +306,16 @@ def apply_file_filter(self, filter):
     def flatten(self, config, all_requests, common_vars):
         return PrintFileRequest(
             name = self.name,
-            output_file = self.output_file,
+            output_file = self.txt_file,
             content = self._generate_index_file(common_vars)
+        ).flatten(config, all_requests, common_vars) + SingleExecutionRequest(
```

Comment:
Please add a line feed after the plus sign.
I suggest indentation like this:
    return
        PrintFileRequest(
            keys = values
        ).flatten(args) +
        SingleExecutionRequest(
            keys = values
        ).flatten(args)